atom/common/lib/asar.coffee

```diff
@@ -87,10 +96,14 @@ overrideAPI = (module, name, arg = 0) ->
     return overrideAPISync module, name, arg unless typeof callback is 'function'
 
     archive = getOrCreateArchive asarPath
-    return callback new Error("Invalid package #{asarPath}") unless archive
+    unless archive
```

Comment:
It is preferred to use `return invalidArchiveError asarPath, callback unless archive` to make lines shorter, since the same line would be repeated for many times.